### ISSUE TITLE
No MainWindow.xaml in project created with proje template

### ISSUE THREAD

**Issue by eliassal** [lepoco/wpfui#338]
I downloaded the extension for VS2022, then created a project based on this extension, I have a solution created named UIDesktopApp1, I can see app.xaml but not the file 
`MainWindow.xaml` 

as indicated in the tutorial

`First,` let's modify MainWindow.xaml

![UIWPF](https://user-images.githubusercontent.com/8447864/183460572-73f80bd0-3641-4201-b9eb-82cc119f522c.jpg)

Thanks for your help

**Comment by pomianowski** [comment]
`Pages/Container.xaml` is a MainWindow

**Comment by eliassal** [comment]
Thanks, I opened it but it cant be displayed, I tried to build but it did not go through

![UIWPF_002](https://user-images.githubusercontent.com/8447864/183592033-6d4a26bb-f4be-42db-8f24-1e7276316230.jpg)

**Comment by pomianowski** [comment]
Errors that can be seen in the console result from incorrect operation of the source generators.  
Generators don't work in some versions of Visual Studio and WPF
https://github.com/dotnet/core/blob/main/release-notes/6.0/known-issues.md#issues-building-wpf-application-with-windows-desktop-607-and-608=

With workaround:
* To enable build in Windows Desktop 6.0.7, navigate to the directory containing the `Microsoft.WinFx.targets` file ( `C:\Program Files\dotnet\sdk\6.0.302\Sdks\Microsoft.NET.Sdk.WindowsDesktop\targets` )
* Add the following target in the file:
```xaml
<Target Name="RemoveDuplicateAnalyzers" BeforeTargets="CoreCompile">
    <ItemGroup>
        <FilteredAnalyzer Include="@(Analyzer->Distinct())" />
        <Analyzer Remove="@(Analyzer)" />
        <Analyzer Include="@(FilteredAnalyzer)" />
    </ItemGroup>
</Target>
```

**Comment by eliassal** [comment]
This did not help. After adding the target in the file, fired Vs 2022, I got 

An error was encountered while opening associated documents the last time this solution was loaded. Document load is being skipped during this solution load in order to avoid that error.

Then when I tried to build the project, got 

![UIWPF_003](https://user-images.githubusercontent.com/8447864/184086268-ef591d36-a8eb-41b0-84e2-98fbd18dd062.jpg)
